ale/vis.py

```diff
@@ -0,0 +1,206 @@
+
+import tempfile
+
+import json
+import knoten
+import pvl
+import pyproj
+import csmapi
+
+from numbers import Number
+
+import numpy as np
+import pandas as pd
+
+from pysis import isis
+from pysis.exceptions import ProcessError
+
+import holoviews as hv
+from holoviews import opts, dim
+from bokeh.models import HoverTool
+
+
+def reproject(record, semi_major, semi_minor, source_proj, dest_proj, **kwargs):
+    """
+    Thin wrapper around PyProj's Transform() function to transform 1 or more three-dimensional
+    point from one coordinate system to another. If converting between Cartesian
+    body-centered body-fixed (BCBF) coordinates and Longitude/Latitude/Altitude coordinates,
+    the values input for semi-major and semi-minor axes determine whether latitudes are
+    planetographic or planetocentric and determine the shape of the datum for altitudes.
+    If semi_major == semi_minor, then latitudes are interpreted/created as planetocentric
+    and altitudes are interpreted/created as referenced to a spherical datum.
+    If semi_major != semi_minor, then latitudes are interpreted/created as planetographic
+    and altitudes are interpreted/created as referenced to an ellipsoidal datum.
+
+    Parameters
+    ----------
+    record : object
+          Pandas series object
+
+    semi_major : float
+              Radius from the center of the body to the equater
+
+    semi_minor : float
+              Radius from the pole to the center of mass
+
+    source_proj : str
+                      Pyproj string that defines a projection space ie. 'geocent'
+
+    dest_proj : str
+                   Pyproj string that defines a project space ie. 'latlon'
+
+    Returns
+    -------
+    : list
+    Transformed coordinates as y, x, z
+
+    """
+    source_pyproj = pyproj.Proj(proj = source_proj, a = semi_major, b = semi_minor)
+    dest_pyproj = pyproj.Proj(proj = dest_proj, a = semi_major, b = semi_minor)
+
+    y, x, z = pyproj.transform(source_pyproj, dest_pyproj, record[0], record[1], record[2], **kwargs)
+
+    return y, x, z
+
+
+def point_info(cube_path, x, y, point_type, allow_outside=False):
```

Comment:
`point_info` and `reproject` are required to get it working but currently exist only in autocnet. I'm working on getting them PRed into `pysis` and `knonten` respectively.